### PR TITLE
8.2.10 Release

### PR DESCRIPTION
**Pull request by su-sws-jira**
# DO NOT DELETE

### CODE REVIEW

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

File: patches/contrib/ui_patterns_views-preview.patch [hunk 1]
```diff
@@ -96,7 +96,7 @@ index 388b287..6f835d4 100644
 +  foreach ($view->field as $id => $field) {
 +    // render this even if set to exclude so it can be used elsewhere.
 +    $field_output = $view->style_plugin->getField($row->index, $id);
-+    $empty = $field->isValueEmpty($field->getValue($row), $field->options['empty_zero']);
++    $empty = $field->getPluginId() == 'field' && $field->isValueEmpty($field->getValue($row), $field->options['empty_zero']);
```
Comment (silverli) [inline patches/contrib/ui_patterns_views-preview.patch:99]: 👍

Comment (silverli) [reply patches/contrib/ui_patterns_views-preview.patch:99]: this works to fix an issue I was seeing when we mapped "Link to Content" to other fields in a pattern view.

previously, the field for link to content would always render as null, even when there was a node detail page to link to.

Screen Shot 2020-06-01 at 5 56 06 PM

**Pull request merged**